openid-sharedsignals-framework-1_0.md

```diff
@@ -1855,31 +1955,60 @@ reason
 
 > OPTIONAL. Provides a short description of why the Transmitter has updated the
   status.
+  
+As with any SSF Event, this event has a top-level `sub_id` claim:
 
-subject
+sub_id
 
-> REQUIRED. Specifies the stream whose status has been updated.
-  The value of the `subject` field
+> REQUIRED. The top-level `sub_id` claim specifies the Subject Principal for whom the status has been updated.
```

Comment:
In #88 we removed the ability to set status for a specific subject. Therefore the "status updated" event MUST always have a `sub_id` that identifies the stream itself, like the verification event